wagtail/core/models.py

```diff
@@ -3604,13 +3602,14 @@ def cancel(self, user=None, resume=False, comment=''):
     def copy(self, update_attrs=None, exclude_fields=None):
         """Copy this task state, excluding the attributes in the ``exclude_fields`` list and updating any attributes to values
         specified in the ``update_attrs`` dictionary of ``attribute``: ``new value`` pairs"""
+        def _save_copy_instance(instance):
+            # Called from self._copy when it's ready to save an instance
+            instance.save()
+            return instance
+
         copy_instance, _ = self._copy(exclude_fields, update_attrs)
```

Comment:
I think you forgot to update the call to `_copy` to pass in `_save_copy_instance` here. (Looks like this gets refactored out completely later on, though, so perhaps that doesn't matter...)

Reply:
yep, https://github.com/wagtail/wagtail/pull/6277/commits/4fea8cd71bc1e8963bc11d9f2224433f4f920766#diff-984f91690294c77757dbe73c72481f9bR3595 is where it would have been removed again.

Reply:
Oh, sorry. I was sort of making this up as I went along! Feel free to squash all this at the end though